### DISCUSSION TITLE
2.3.2 Issues

### DISCUSSION THREAD

**Discussion by tlinsenmeyer** [Security-Onion-Solutions/securityonion#1838] [Unsupported Versions] [answered]
I do not have a problem at all with 2.3.1 but as soon as I upgrade or fresh install the manager and sensor, Security Onion creates a constant external network connection, Web site does not load and all certs from the sensor to the manager are invalid.  I would just stay on 2.3.1 but my sensor server does not support uefi/gpt format only mbr, and so far I cannot find an iso that is mbr.

**Comment by TOoSmOotH** [comment, marked as answer]
The ISO supports both UEFI and BIOS boot. Make sure you change your bios to legacy or bios boot before the install and it should detect BIOS mode.

**Reply by tlinsenmeyer** [reply]
I am using Rufus to put the ISO on my drive, I cannot change the partition table and my server does not see a MBR to boot from, what else can I do?

**Reply by tlinsenmeyer** [reply]
My sensor server is old and does not support UEFI

**Reply by TOoSmOotH** [reply]
You can use the centos 7 minimal install and do a network install.
